Tests/Integration/DSCResources/IisServer.integration.tests.ps1

```diff
@@ -18,18 +18,29 @@ try
 
     foreach ($stig in $stigList)
     {
-        [xml] $powerstigXml = Get-Content -Path $stig.Path
+        $orgSettingsPath = $stig.Path.Replace('.xml', '.org.default.xml')
+        $blankSkipRuleId = Get-BlankOrgSettingRuleId -OrgSettingPath $orgSettingsPath
+        $powerstigXml = [xml](Get-Content -Path $stig.Path) |
+            Remove-DscResourceEqualsNone |
+            Remove-SkipRuleBlankOrgSetting -OrgSettingPath $orgSettingsPath
```

Comment:
This should be indendented or put on the line above it.

Reply:
Done